app/src/main/java/com/cornellappdev/uplift/networking/UpliftAuthRepository.kt

```diff
@@ -0,0 +1,60 @@
+package com.cornellappdev.uplift.networking
+
+import android.util.Log
+import com.apollographql.apollo3.ApolloClient
+import com.cornellappdev.uplift.CreateUserMutation
+import com.google.firebase.firestore.auth.User
+//import com.cornellappdev.uplift.android.model.api.CreateUserBody
+//import com.cornellappdev.uplift.android.model.api.LoginBody
+//import com.cornellappdev.uplift.android.model.api.RetrofitInstance
+//import com.cornellappdev.resell.android.model.api.User
+//import com.cornellappdev.resell.android.model.core.UserInfoRepository
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class UpliftAuthRepository @Inject constructor(
+  private val userInfoRepository: UserInfoRepository
+) {
+  private val apolloClient = ApolloClient.Builder()
+//        .serverUrl("https://uplift-backend.cornellappdev.com/graphql")
+    .serverUrl("https://uplift-dev.cornellappdev.com/graphql")
+    .build()
```

Comment:
This is an outdated and dangerous way of referencing the backend URL--see `UpliftApiRepository` for how I implemented a better (and more correct) way of doing this